apps/nextjs-website/src/app/auth/sign-up/page.tsx

```diff
@@ -16,6 +16,7 @@ import { Auth } from 'aws-amplify';
 import { useRouter, useSearchParams } from 'next/navigation';
 import { useCallback, useState } from 'react';
 import { SignUpUserData } from '@/lib/types/sign-up';
+import { LoaderPhase } from '@/lib/types/loader';
 
```

Comment:
I'd add a const relative to timeout to add some context and avoid magic numbers
```suggestion
const RESET_SEND_EMAIL_BUTTON_AFTER_MS = 4000;

```